bentoml/__init__.py

```diff
@@ -26,18 +26,15 @@
 # Configuring logging properly before loading other modules
 configure_logging()
 
-from bentoml.saved_bundle import load_from_dir, save_to_dir  # noqa: E402
-from bentoml.service import (  # noqa: E402
-    BentoService,
-    api_decorator as api,
-    env_decorator as env,
-    web_static_content_decorator as web_static_content,
-    artifacts_decorator as artifacts,
-    ver_decorator as ver,
-    save,
-)
-
 from bentoml.cli import create_bentoml_cli
+from bentoml.saved_bundle import load_from_dir, save_to_dir  # noqa: E402
```

Comment:
will this affect the import speed of bentoml, since this is related to https://github.com/bentoml/BentoML/pull/1608

Reply:
You mean this line of code or this commit?

Reply:
by this commit

Reply:
How could this commit affect the importing speed, would you mind sharing the detail?

Reply:
actually this PR improve the import time

this PR:
https://gyazo.com/9159b63feb4ab1c3c65e1ff7eeb0c502

master branch
https://gyazo.com/d991564d4abc540266b60cf6965e7e56

How to test it:

```bash
python -mcProfile -o program.prof bentoml/artifacts/__init__.py
```
Run the runtime profiler with [tuna](https://github.com/nschloe/tuna):

```bash
tuna program.prof
```